api/pyproject.toml

```diff
@@ -35,3 +35,194 @@ skip = ['migrations','.venv','.direnv']
 
 [tool.pytest.ini_options]
 addopts = ['--ds=app.settings.test', '-vvvv', '-p', 'no:warnings', '-n', 'auto']
+
+[tool.poetry]
+name = "y"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
```

Comment:
```suggestion
python = "~3.10,~3.11"
```

Reply:
Is 3.10 officially supported, the docker build failed after adding 3.10?